### DISCUSSION TITLE
CheckFailedNotification config can only process one type

### DISCUSSION THREAD

**Discussion by mgkimsal** [spatie/laravel-health#88] [Q&A]
In config/health.php, setting

```
CheckFailedNotification::class => ['mail', 'slack'] 
```

only 'mail' will be processed. No errors in logs, nothing I can see is 'wrong', just slack message isn't processed.

Swapping them around

```
CheckFailedNotification::class => ['slack', 'mail'] 
```

means a slack web hook message works, but mail doesn't.

It's marked as an array.  Is it supposed to be able to handle multiple?  The docs don't indicate it, but the array syntax makes me think it was intended to work that way.

**Comment by freekmurze** [comment]
The notification channels are being determined here: https://github.com/spatie/laravel-health/blob/720f2477fa5c5d8f31ae456966633035f99b1384/src/Notifications/CheckFailedNotification.php#L20-L27

You could try to debug by dumping what the `via` returns, and whether `toSlack` is being called.

Did you already try to `php artisan clear:config`?

If you find that he package is in error, please send a PR 👍

**Reply by moisish** [reply]
I have opened a PR here https://github.com/spatie/laravel-health/pull/107
